lib/exercism/comment.rb

```diff
@@ -36,4 +36,16 @@ def submission_user
   def mentions
     ExtractsMentionsFromMarkdown.extract(body)
   end
+
+  def qualifying?
+    submission_user != user
+  end
+
+  def increment_five_a_day
+    if FiveADayCount.exists?(:user_id => user.id)
+      FiveADayCount.where(:user_id => user.id).first.increment!(:total)
```

Comment:
Ah, right. I guess `increment` is a method on the instance, not on the relation. OK, then :)